abstio/src/abst_data.rs

```diff
@@ -48,9 +48,29 @@ impl Manifest {
                 // Always grab all of these
                 continue;
             }
-            if path.starts_with("data/input/shared") && !data_packs.input.is_empty() {
-                // Grab all of these if the user has opted into any input data at all
-                continue;
+            // If the user has opted into any input data at all, we want to grab some of the shared
+            // input files.
+            if path.starts_with("data/input/shared") {
+                // But some of the files are large, so of course we hardcode some more overrides
+                // here. Maybe some of this data should be scoped to a country, not a city.
+                if path.ends_with("Road Safety Data - Accidents 2019.csv")
+                    || path.ends_with("wu03ew_v2.csv")
+                    || path.ends_with("zones_core.geojson")
+                {
+                    if data_packs.input.iter().any(|x| x.starts_with("gb/")) {
```

Comment:
What's the goal of this logic - to skip the download if we're in GB?

Reply:
Inverted, this is a `filter` method -- `continue` means we leave the entry alone and so we will download it. These 3 files are for any GB map.